sycl/test-e2e/EnqueueNativeCommand/custom-command-hip.cpp

```diff
@@ -1,6 +1,6 @@
 // FIXME: the rocm include path and link path are highly platform dependent,
 // we should set this with some variable instead.
-// RUN: %{build} -o %t.out -I%rocm_path/include -L%rocm_path/lib -lamdhip64
+// RUN: %{build} -Wno-error=deprecated-pragma -o %t.out -I%rocm_path/include -L%rocm_path/lib -lamdhip64
```

Comment:
I wonder why "deprecated-pragma" is passed, when __AMDGCN_WAVEFRONT_SIZE is a macro?

Reply:
Good question, the error message was `-Werror, -Wdeprecated-pragma` in test. Not sure which clang change caused that.

Reply:
Sure. https://github.com/intel/llvm/issues/16395 to track.